packages/gradle.properties

```diff
@@ -31,3 +31,15 @@ android.useAndroidX=true
 # Allow the default dependency name to match the client debug build type. Otherwise the client project has to
 # explicitly specify the debug variant (ex: implementation("io.realm.kotlin:library-base-debug:0.8.0"))
 kotlin.android.buildTypeAttribute.keep=false
+
+# testRepository - will control how the gradle project is configured. The behavior overall falls in
+# two categories.
+# - If unset or the empty string the project is setup for local development where tests
+#   (test-base/test-sync) is executed against sub-projects
+#testRepository=
+# - If set then tests (test-base/test-sync) are executed against the artifacts in the specified
```

Comment:
We should probably also mention that doing this will disable all non-test modules.